hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -269,7 +269,6 @@ message GetScmInfoResponseProto {
     required string clusterId = 1;
     required string scmId = 2;
     repeated string peerRoles = 3;
-    optional bool scmRatisEnabled = 4;
```

Comment:
This doesn't seem to be backwards compatible.  While there were no releases with HDDS-11268 (where this was added few months ago), someone with custom build from `master` could already be using it.  I would prefer to keep it.